libs/hmpb/src/ballot_components.tsx

```diff
@@ -196,6 +224,27 @@ export function Bubble({
   );
 }
 
+export function BubbleWrapper({
+  optionInfo,
+  isWriteIn,
+}: {
+  optionInfo: OptionInfo;
+  isWriteIn?: boolean;
+}): React.ReactElement {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        // Match line-height of text to align bubble to center of first line of option label or write-in candidate name
+        height: isWriteIn ? '1.25rem' : '1.2rem',
```

Comment:
It seems like a bit of a leaky abstraction to have this height defined here. Maybe it should be defined by the consumer (either via `styled-components` or a `style` prop)?